wasp_eureka/client.py

```diff
@@ -242,13 +234,19 @@ def __init__(self,
         url = self._eureka_url + path
         logger.debug('Performing %s on %s with payload: %s', method, path,
                      data)
-        async with _SESSION.request(method, url, data=data) as resp:
-            if 400 <= resp.status < 600:
-                # noinspection PyArgumentList
-                raise EurekaException(HTTPStatus(resp.status),
-                                      await resp.text())
-            logger.debug('Result: %s', resp.status)
-            return await resp.json()
+        headers = {
+            # They default to using XML.
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+        }
+        async with ClientSession(headers=headers) as session:
```

Comment:
The reason we didn't move this in here is because it sacrifices connection pooling, so we'd want to set the client somewhere where it can be re-used.

Instead it's probably better to set this on the instance in `__init__` since we allow a param for the event loop (or can request it via `asyncio.get_event_loop()`